libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -233,16 +233,16 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     explorerViews: [],
   },
-  avalanche: {
-    type: "CryptoCurrency",
-    id: "avalanche",
-    coinType: 9000,
-    name: "Avalanche",
-    managerAppName: "Avalanche",
-    ticker: "AVAX",
-    scheme: "avalanche",
+  avalanchecchain: {
+    type: "CryptoCurrency",
+    id: "avalanchecchain",
+    coinType: 60,
```

Comment:
proposal: this could be an enum to make sure devs add values that actually make sense

```ts
// ledgerjs/packages/types-cryptoassets/src/index.ts
enum CoinType {
   ETH = 60,
   ...
}

interface CryptoCurency {
     ...,
    coinType: CoinType
}
``

```ts
// libs/ledgerjs/packages/cryptoassets/src/currencies.ts
export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
 ...
 avalanchecchain: {
    type: "CryptoCurrency",
    id: "avalanchecchain",
    coinType: CoinType.ETH,
    ....
  },
}
```